Source/WebCore/Modules/mediastream/libwebrtc/LibWebRTCStatsCollector.cpp

```diff
@@ -379,10 +459,12 @@ static inline void fillRTCIceCandidateStats(RTCStatsReport::IceCandidateStats& s
     if (rtcStats.protocol.is_defined())
         stats.protocol = fromStdString(*rtcStats.protocol);
 
-    if (rtcStats.candidate_type.is_defined())
-        stats.candidateType = iceCandidateState(*rtcStats.candidate_type);
+    ASSERT(rtcStats.candidate_type.is_defined());
+    auto candidateType = iceCandidateState(*rtcStats.candidate_type);
+    ASSERT(candidateType);
```

Comment:
We might want to modify iceCandidateState to return a RTCIceCandidateType instead of an optional